packages/react-app/src/App.jsx

```diff
@@ -1,13 +1,6 @@
 import { Button, Col, Menu, Row } from "antd";
 import "antd/dist/antd.css";
-import {
```

Comment:
I think you might have a custom prettier/formatting settings in your IDE (different from `packages/react-app/.prettierrc`), because I see some formatting rules not defined in the `.prettierrc` file.

E.g. this line won't go in a single line because the length would be more than > 120 chars (rule: `"printWidth": 120` in the `.prettierrc` file.